preview/MsixCore/msixmgr/msixmgr.vcxproj

```diff
@@ -162,8 +172,13 @@
       <AdditionalDependencies>msixmgrlib.lib;msix.lib;msi.lib;comctl32.lib;gdiplus.lib;version.lib;taskschd.lib;windowsapp_downlevel.lib;windowsapp.lib;urlmon.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <DelayLoadDLLs>api-ms-win-core-winrt-string-l1-1-0.dll;api-ms-win-core-winrt-l1-1-0.dll;api-ms-win-core-winrt-error-l1-1-0.dll;api-ms-win-core-winrt-error-l1-1-1.dll</DelayLoadDLLs>
     </Link>
-    <PostBuildEvent>
-      <Command>xcopy /Y /D "..\..\..\.vs\bin\msix.dll" "$(OutDir)"</Command>
+    <PostBuildEvent Condition="$(RunOrNot)=='true'">

```

Comment:
>RunOrNot [](start = 33, length = 8)

too generic of a name; name it like RunPostbuild or something like that.

Reply:
Backed out these changes.